support-frontend/assets/pages/subscriptions-landing/copy/subscriptionCopy.tsx

```diff
@@ -61,19 +61,29 @@ const getDisplayPrice = (
 	return `${currency}${fixDecimals(price)}/${billingPeriod}`;
 };
 
-const getDigitalEditionPrices = (countryGroupId: CountryGroupId): string => {
+const getDigitalEditionPrice = (
+	countryGroupId: CountryGroupId,
+	billingPeriod: BillingPeriod,
+): string => {
 	const currencyKey = detect(countryGroupId);
 	const currency = currencies[currencyKey].glyph;
 	const product = productCatalog['DigitalSubscription'];
-	const priceMonthly = product?.ratePlans[Monthly]?.pricing[currencyKey];
-	const currencyPriceMonthly = priceMonthly
-		? `${currency}${fixDecimals(priceMonthly)}/${Monthly}`
-		: '';
-	const priceAnnual = product?.ratePlans[Annual]?.pricing[currencyKey];
-	const currencyPriceAnnual = priceAnnual
-		? ` ${currency}${fixDecimals(priceAnnual)}/${Annual}`
-		: '';
-	return `${currencyPriceMonthly}${currencyPriceAnnual}`;
+	const price = product?.ratePlans[billingPeriod]?.pricing[currencyKey];
+	if (!price) {
+		return '';
+	}
+	const fixedPriceWtihCurrency = `${currency}${fixDecimals(price)}`;
+	const billingFrequency =
+		billingPeriod === Annual ? 'Annually' : billingPeriod;
+	return `${fixedPriceWtihCurrency}/${billingFrequency}`;
+};
+const getDigitalEditionPrices = (countryGroupId: CountryGroupId): string => {
+	const priceMonthly = getDigitalEditionPrice(countryGroupId, Monthly);
+	const priceAnnual = `${priceMonthly ? ' or' : ''}${getDigitalEditionPrice(
+		countryGroupId,
+		Annual,
+	)}`;
+	return `${priceMonthly}${priceAnnual}`;
```

Comment:
Gotcha, so if the 'or' is based on the existence of both monthly and anual prices I would be explicit about it, and not use priceAnnual for that. what do you think about:
```suggestion
	const priceMonthly = getDigitalEditionPrice(countryGroupId, Monthly);
	const priceAnnual = getDigitalEditionPrice(countryGroupId,Annual);
	
	return [priceMonthly, priceAnnual].join(' or ');
```

Reply:
I agree with @maloky that keeping the 'or' out of `priceAnnual` as it's easier to read, however I think we could just return `${priceMonthly} or ${priceAnnual}` rather that using the array join?

Reply:
@maloky : Nice and neat solution. Thanks, have applied it 👌